lib/active_fulfillment/services/amazon_mws.rb

```diff
@@ -450,5 +453,17 @@ def build_status_request
     def escape(str)
       CGI.escape(str.to_s).gsub('+', '%20')
     end
+
+    private
+
+    def secure_compare(a, b)
+      return false unless a.bytesize == b.bytesize
+
+      l = a.unpack "C#{a.bytesize}"
```

Comment:
This code comes from http://api.rubyonrails.org/classes/ActiveSupport/SecurityUtils.html#method-c-secure_compare, which unfortunately is only available starting in ActiveSupport 4.2.1. Someday when we no longer support older versions, we can just call it directly.